.github/workflows/ci.yaml

```diff
@@ -199,3 +192,11 @@ jobs:
           cmake .
           make
           ./cpp-tests
+      - name: Setup iOS build
+        if: matrix.os == 'macos-10.15'
```

Comment:
Is there a way to have this just check for "macos"? It might be nice to not have to update all the specific versions everywhere if we update.

Reply:
Good idea. I made this change in ed77912dfe334add627dd3142755ec00212e91b0.